Makefile

```diff
@@ -1797,16 +1799,30 @@ $(ESP_COMBINED512): $(USER1_BIN) $(USER2_BIN)
 # IMPORTANT: this only works if DISABLE_LTO is defined, e.g. `DISABLE_LTO=1 make`
 topstrings: $(PARTIAL)
 	$(Q)for f in `find . -name \*.o`; do \
-	  str=$$($(OBJDUMP) -j .rodata.str1.4 -h $$f 2>/dev/null | \
-	    egrep -o 'rodata.str1.4 [0-9a-f]+' | \
+	  str=$$($(OBJDUMP) -j .rodata.str1.1 -j .rodata.str1.4 -h $$f 2>/dev/null | \
+	    egrep -o 'rodata.str1.. [0-9a-f]+' | \
 	    awk $$(expr match "$$(awk --version)" "GNU.*" >/dev/null && echo --non-decimal-data) \
 	      -e '{printf "%d\n", ("0x" $$2);}'); \
 	  [ "$$str" ] && echo "$$str $$f"; \
 	done | \
 	sort -rn >topstrings
 	$(Q)echo "Top 20 from ./topstrings:"
 	$(Q)head -20 topstrings
-	$(Q)echo "To get details: $(OBJDUMP) -j .rodata.str1.4 -s src/FILENAME.o"
+	$(Q)echo "To get details: $(OBJDUMP) -j .rodata.str1.1 -j .rodata.str1.4 -s src/FILENAME.o"
+
+# Same as topstrings but consider all read-only data
+topreadonly: $(PARTIAL)
```

Comment:
I'm a bit lost, what is the goal here?

My goal was to get a bit beyond the 1400 jsvars and more heap so I could load programs that I currently can't run reliably and so that the thing wouldn't run out of heap as soon as an extra buffer is needed due to a network hiccup.With the jswsym change that has been fully accomplished.